components/function-controller/internal/controllers/serverless/suite_test.go

```diff
@@ -20,7 +20,11 @@ import (
 )
 
 const (
-	testNamespace = "test-namespace-name"
+	testNamespace  = "test-namespace-name"
+	fakeDockerfile = `ARG base_image={{ include "imageurl" (dict "reg" .Values.global.containerRegistry "img" .Values.global.images.function_runtime_python39) }}
```

Comment:
We can use normal name instead of helm templating

Reply:
Changed to `some_image`